src/main/java/com/projetogerenciamentocurso/gerenciamentocurso/dto/AlunoDto.java

```diff
@@ -0,0 +1,27 @@
+package com.projetogerenciamentocurso.gerenciamentocurso.dto;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.persistence.Id;
+
+import com.projetogerenciamentocurso.gerenciamentocurso.models.Pessoa;
+import com.projetogerenciamentocurso.gerenciamentocurso.models.Turma;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode(callSuper=false)
+public class AlunoDto extends Pessoa{

```

Comment:
Pessoa é uma classe de domínio. Ela não deve ser classe mãe de um DTO. Deve ser criado o PessoaDTO para isso.
```suggestion
public class AlunoDto extends PessoaDto{
```

Reply:
feito